sagemaker-clarify/bias-and-explainability/student_performance_clarify.ipynb

```diff
@@ -0,0 +1,858 @@
+{
```

Comment:
<p>Add the UCI ML repository as a reference. </p><blockquote>Dua, D. and Graff, C. (2019). UCI Machine Learning Repository [http://archive.ics.uci.edu/ml]. Irvine, CA: University of California, School of Information and Computer Science.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2916/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>